sources/gov/kaunas/educational_institutions.yml

```diff
@@ -2,10 +2,7 @@ id: "gov/kaunas/municipality/educational_institutions"
 title: "Educational institutions"
 description: "CSV file containing data about Kaunas educational institutions"
 type: "source"
-source:
-  - "https://github.com/Kauno-miestas/Svietimas/blob/master/Svietimo%20istaigos.csv"
-  - type: '#/definitions/csv'
-    sep: ","
+source: "csv+https://github.com/Kauno-miestas/Svietimas/blob/master/Svietimo%20istaigos.csv"
```

Comment:
Link should point to raw csv file, not to github page with html table generated from raw csv file.